modules/migration-viewing-migration-plan-resources.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * migrating_from_ocp_3_to_4/troubleshooting-3-4.adoc
+// * migration-toolkit-for-containers/troubleshooting-mtc.adoc
+
+[id="migration-viewing-migration-plan-resources_{context}"]
+= Viewing migration plan resources
+
+You can view the resources of a migration plan to monitor the progress of a live migration or to troubleshoot a failed migration. You use the {mtc-short} web console to copy a command to your clipboard. Then, you run the command from the command line interface (CLI).
+
+If you are troubleshooting a failed migration, start with a high-level resource that has failed and then work down the resource tree, towards the lower-level resources.
+
+.Procedure
+
+. In the {mtc-short} web console, click *Migration Plans*.
+. Click the *Migrations* number next to a migration plan to view the *Migrations* page.
++
+The *Migrations* page displays the migration types associated with the migration plan, for example, *Stage* or *Cutover* for warm migration.
+
+. Click the *Type* to view the *Migration details* page.
++
+The *Migration details* page displays the steps of the migration and their status.
+
+. Expand *Migration resources* to view the migration resources and their status.
+. Click the {kebab} next to a resource and select one of the following options:
+
+* *Copy `oc describe` command* copies the command to your clipboard. Log in to the relevant cluster and run the command from the CLI. The command displays the conditions and events of the resource as a YAML file.
+* *Copy `oc logs` command*: copies the command to your clipboard. Log in to the relevant cluster and run the command from the CLI. The command displays the logs of the resource. If the resource supports log filtering, the filtered log is displayed.
+* *View JSON*: The conditions and events of the resource are displayed as a JSON file in a web browser.
```

Comment:
JSON content shows up the resource with json formal, it's similar to what `oc get` command. I think this description is not accurate